packages/firebase_crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0+5
+
+* Fix example app build.
```

Comment:
I would say what changed in the changelog to make it work, or at least what the error that was fixed is.